lib/engine/step/special_track.rb

```diff
@@ -16,16 +16,8 @@ def actions(entity)
         ACTIONS
       end
 
-      def description
-        "Lay Track for #{@company.name}"
-      end
-
-      def active_entities
-        @company ? [@company] : super
-      end
-
       def blocks?
-        tile_lay_abilities(@company)&.blocks
+        false
```

Comment:
is this gonna break stuff?

Reply:
I don't think that line actually worked lol. Traced the line back to a commit to a fix 1846 special tile lays. I tested the 1846 special tile lay privates by hand and it all worked great. Cases that should fail, failed. Cases that should work, worked.

Reply:
Just found that this did break stuff for 18Chesapeake. C-P and B&S are supposed to block after their first tile is laid, and now they don't. The abilities don't set `blocks` explicitly, but it defaults to `true` in `engine/ability/tile_lay.rb`